editor/src/messages/tool/tool_messages/path_tool.rs

```diff
@@ -116,6 +132,10 @@ struct PathToolData {
 	previous_mouse_position: DVec2,
 	alt_debounce: bool,
 	opposing_handle_lengths: Option<OpposingHandleLengths>,
+	drag_start: ViewportPosition,
+	drag_current: ViewportPosition,
```

Comment:
Please use the existing `drag_start_pos` and `previous_mouse_position` to avoid duplicate fields here.